inventory

```diff
@@ -1 +1,9 @@
-[sigstore]
\ No newline at end of file
+[sigstore]
+<VM_IP_ADDRESS>
+
+[sigstore:vars]
+ansible_password=<ROOT_PASSWORD_VM>
+ansible_user=redhat
+remote_user=root
+become=true
+become_user=root
```

Comment:
clearly I added everything I knew to run this - i'll check on what's necessary here :)